tools/db_bench_tool.cc

```diff
@@ -4672,8 +4685,18 @@ class Benchmark {
       mgr =
           std::make_shared<RoundRobinManager>(GetBuiltinV2CompressionManager());
     } else if (!strcasecmp(FLAGS_compression_manager.c_str(),
-                           "costpredictor")) {
-      mgr = CreateCostAwareCompressionManager();
+                           "autotunecompressor")) {
+      auto ratelimiter_throughput = FLAGS_rate_limiter_bytes_per_sec;
+      double io_upper_bound = FLAGS_autotune_iogoal * ratelimiter_throughput;
+      double io_lower_bound = FLAGS_autotune_miniogoal * ratelimiter_throughput;
+      double cpu_upper_bound = FLAGS_autotune_cpubudget;
+      double cpu_lower_bound = FLAGS_autotune_mincpubudget;
```

Comment:
Can we use upper bound and lower bound for the FLAG name too? "min goal" doesn't make sense to me